package.json

```diff
@@ -38,7 +41,8 @@
     "url": "https://github.com/gnosis/dex-contracts/issues"
   },
   "homepage": "https://github.com/gnosis/dex-contracts#readme",
-  "dependencies": {
+  "dependencies": {},
```

Comment:
Can we update the README and CI scripts to indicate that we now need to do npm install dev dependencies?

Reply:
AFAIK `npm install` installs all dependencies including dev dependencies as far as I know. As for CI, Travis runs `npm ci` for installing dependencies which also installs all dev dependencies. From what I understand vev dependencies only do not get installed when `dex-contract` is itself used as a dependency as an NPM package (which assumes the package was already built and the dev dependencies are no longer needed).